src/js/server/models/Hacker.js

```diff
@@ -184,15 +193,26 @@ Hacker.upsertAndFetchFromMlhUser = function (mlhUser) {
 };
 
 // Determine the headline application status
-Hacker.deriveOverallStatus = function (applicationStatus, responseStatus, teamApplicationStatus, furtherDetailsStatus) {
+Hacker.deriveOverallStatus = function (applicationStatus, responseStatus, teamApplicationStatus, rsvpStatus) {
   if (applicationStatus == statuses.application.INCOMPLETE || teamApplicationStatus == statuses.application.INCOMPLETE)
     return statuses.overall.INCOMPLETE;
   else if (responseStatus == statuses.response.PENDING)
     return statuses.overall.IN_REVIEW;
   else if (responseStatus == statuses.response.REJECTED)
     return statuses.overall.REJECTED;
-  else if (furtherDetailsStatus == statuses.furtherDetails.INCOMPLETE)
-    return statuses.overall.ACCEPTED_INCOMPLETE;
-  else if (furtherDetailsStatus == statuses.furtherDetails.COMPLETE)
-    return statuses.overall.ACCEPTED_COMPLETE;
+  else if (rsvpStatus == statuses.rsvp.INCOMPLETE)
+    return statuses.overall.INVITED_AWAITING_RSVP;
+  else if (rsvpStatus == statuses.rsvp.COMPLETE_NO)
+    return statuses.overall.INVITED_DECLINED;
+  else if (rsvpStatus == statuses.rsvp.COMPLETE_YES)
+    return statuses.overall.INVITED_COMPLETE;
```

Comment:
Should we make their invited status be dependent on an existence of a ticket and not a yes RSVP? There is not a 1:1 mapping between the two